web-frontend/src/pages/SelectAccount/SelectAccount.tsx

```diff
@@ -41,10 +41,15 @@ export default function SelectAccount() {
                 to={routeFor('home')}
                 state={{ nickname: account.nickname }}
               >
-                <div className="mb-4" key={account.nickname}>
+                <div className="mb-4 theme-light" key={account.nickname}>
```

Comment:
Ohh I see! That's not good because we are changing the whole css template just for a button. We must fix that in ui-kit (with variants like in buttons) and then use it here.

I'll take care of that!

Reply:
I think it's ok here to set the theme-light for a div. This will apply light theme to all nested elements and that's what we want. I don't think we need to modify the ui-kit to add a props for the theme.

Reply:
No! It's an error on the ui-kit. We must fix there first. It was inverted. There's no reason to wrap the button in a div to fix something wrong we did there.

Reply:
> The wrap was already there.

It's a mistake! Should not be there!

> I just change the theme on this div because I want the elements inside the div to be light.

Yep! I understand! But is not the right way to do because it's a ui-kit fix that we should care. That's what we'll fix.

Reply:
> It's a mistake! Should not be there!

I think the div wraps to add the margin with `mb-4`

> Yep! I understand! But is not the right way to do because it's a ui-kit fix that we should care. That's what we'll fix.

I don't understand the bug you will fix.

Reply:
> I think the div wraps to add the margin with mb-4

No worries to put a mb-4 in the div, even we can do it direct in the Selector. However set the whole theme (light) to change only the component style seems wrong. The theme should be set only ONCE or per user request. Not on the fly to change the  component style.

Reply:
> No worries to put a mb-4 in the div, even we can do it direct in the Selector. However set the whole theme (light) to change only the component style seems wrong. The theme should be set only ONCE or per user request. Not on the fly to change the component style.

Lucas UI/UX spec says we can switch theme to emphasize a part of the UI. It means, if user chose light mode, this component must be in dark mode. The same way, if user chose dark mode (default choice) then this component must be light. 

So, to make it correct we should set the theme of that component = the opposite current theme of the page. Because the user can't choose de theme right now, I believe we are good.

The reason why I think the component is correct and doesn't need modification is that it's not impossible that we want to use that component with the same theme as the page's theme.

Reply:
I have created issue https://github.com/massalabs/ui-kit/issues/144

Reply:
Closing here since we clarified in the call!